package.nls.json

```diff
@@ -480,6 +480,7 @@
     "DataScience.interactiveWindowModeBannerSwitchAlways": "Always",
     "DataScience.interactiveWindowModeBannerSwitchNo": "No",
     "DataScience.ipykernelNotInstalled": "IPyKernel not installed into interpreter {0}",
+    "DataScience.needIpykernel6": "Ipykernel 6 is needed for debugging, click Install to continue. Or you can run 'pip install ipykernel==6.0.3/conda install ipykernel=6'",
```

Comment:
Is there a better command line to recommend? I feel like pip install --upgrade ipykernel would work too? Or maybe that doesn't always work?

Reply:
But at least the 6.0.3 version seems rather specific

Reply:
I tested this a lot. Unfortunately, --upgrade didn't upgrade mine from 5.5.5 to 6

Reply:
I have an issue on the python extension to fix the installer. If that's fixed, I'll remove the commands from the message.
https://github.com/microsoft/vscode-python/issues/16937